### PR TITLE
Clean segments as releasing collection

### PR DESCRIPTION
**Pull request by yah01**
Signed-off-by: yah01 <yang.cen@zilliz.com>
#17952

### CODE REVIEW

**Comment by yah01** [comment]
rerun ut

**Comment by yah01** [comment]
rerun ut

File: internal/querycoord/task.go [hunk 1]
```diff
@@ -1922,7 +1929,7 @@ func (lbt *loadBalanceTask) processNodeDownLoadBalance(ctx context.Context) erro
 			watchDmChannelReqs := make([]*querypb.WatchDmChannelsRequest, 0)
 			collectionInfo, err := lbt.meta.getCollectionInfoByID(collectionID)
 			if err != nil {
-				log.Error("loadBalanceTask: get collectionInfo from meta failed", zap.Int64("collectionID", collectionID), zap.Error(err))
+				log.Error("loadBalanceTask: get collectionInfo from meta failed", zap.Int64("taskID", lbt.getTaskID()), zap.Int64("collectionID", collectionID), zap.Error(err))
 				lbt.setResultInfo(err)
 				return err
```
Comment (congqixia) [inline internal/querycoord/task.go:1934]: If found some error collection(maybe some trash data in etcd) may leads to all nodedown task fail.

**Comment by yah01** [comment]
rerun ut

**Review by congqixia — APPROVED** [review]
/lgtm

**Pull request merged**